### ISSUE TITLE
Reorganize analytics docs.

### ISSUE THREAD

**Issue by cramforce** [ampproject/docs#238]
I find that https://www.ampproject.org/docs/reference/components/amp-analytics is currently very hard to parse. Breaking it up into multiple pages would probably help a lot:
1. General overview explaining that one can either configure a vendor or roll a one-off config
2. List of all the vendors (important that this is easy to find)
3. How to configure the vendors in general.
4. How to make fully custom configs

**Comment by cramforce** [comment]
@bpaduch What does "Type: IA" mean? :)

Just looked at the guide again, and it could really use from some quick updates to make it in any way comprehensible :)

**Comment by bpaduch** [comment]
@cramforce - IA = Information Architecture.  I'll have a look at this.

**Comment by bpaduch** [comment]
All changes live.
